standard/bridge-v1/cmd/relayer/main.go

```diff
@@ -124,7 +117,6 @@ func main() {
 		Commands: []*cli.Command{{
 			Name:   "start",
 			Usage:  "Start standard bridge relayer",
-			Before: altsrc.InitInputSourceWithContext(flags, altsrc.NewYamlSourceFromFlagFunc(optionConfig.Name)),
```

Comment:
Seems easier to enforce relayer is configured via environment variables. New code was having trouble ingesting the yml